src/main/java/com/endava/client/Client.java

```diff
@@ -0,0 +1,44 @@
+package com.endava.client;
```

Comment:
You have to update your Client to use new endpoint format

Reply:
Updated Client :)